cmd/juju/status/status.go

```diff
@@ -377,6 +383,28 @@ func (c *statusCommand) Run(ctx *cmd.Context) error {
 		}
 		ctx.Infof("Nothing matched specified filter%v.", plural())
 	}
+
+	return nil
+}
+
+func (c *statusCommand) Run(ctx *cmd.Context) error {
+	defer c.close()
+
+	err := c.runStatus(ctx)
+	if err != nil || c.watch == 0 {
+		return err
+	}
+
+	// repeat the call using a ticker
+	ticker := time.NewTicker(c.watch)
+
+	for range ticker.C {
+		err := c.runStatus(ctx)
```

Comment:
The one usability problem with this vs `watch --color juju status --color` is that it just scrolls more and more on the screen, rather than what `watch` does (by putting a fresh screen on each refresh, etc).

So I'm tempted to keep the command line arg, but it actually forks out to `watch -n X --color` (if juju status itself is already in color).

Reply:
That's true. If we want, we can replicate the watch behaviour using ANSI scapes like `fmt.Printf("\033[2J")` to clear the screen.